src/index.html

```diff
@@ -1,13 +1,375 @@
+            </a>
+          </div>
+          <h1 class="header__title">Miami<br>Condo Kings</h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-links">
+            <a href="#home" class="icon icon--cross">
+            </a>
+            
+            <a href="#home" class="logo">
+              <img 
+              src="./images/logo.png"
+              alt="miami logo" 
+              class="logo__img">
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+          <a href="tel:+1 234 555-55-55" 
+             class="menu__call menu__call--phone">+1 234 555-55-55
+          </a>
+
+          <a href="tel:+1 234 555-55-55" 
+             class="menu__call menu__call--call">call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+    <main class="main">
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">Miami best<br>Real estate
+                <a href="#home" 
+                  class="about__link link" 
+                  target="_blank">here
+                </a>
+              </h2>
+            </div>
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__slogan link"
+              target="_blank"
+              >Over 50 years experience in love.</a>
+              
+              <p class="about__text">Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.<br><br>
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.<br><br>
+                <span class="about__text--em">Will never be late!<br>Guaranteed secure transaction - 100%</span>
+              </p>
```

Comment:
```suggestion
              <p class="about__text">
                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.<br><br>
                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.<br><br>
                <span class="about__text--em">
                  Will never be late!<br>Guaranteed secure transaction - 100%
                </span>
              </p>
```